extensions/asyncapi/package.json

```
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "asyncapi",
+  "title": "AsyncAPI",
+  "description": "AsyncAPI extension for Raycast brings some AsyncAPI tools to you.",
+  "icon": "command-icon.png",
+  "author": "smoya",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "validate",
+      "title": "Validate AsyncAPI Document",
+      "description": "Validates a given AsyncAPI document",
+      "mode": "view"
+    },
+    {
+      "name": "validate-clipboard",
+      "title": "Validate AsyncAPI Document",
+      "subtitle": "From clipboard",
+      "description": "Validates an AsyncAPI document from clipboard",
+      "mode": "no-view"
+    },
+    {
+      "name": "open-in-studio",
+      "title": "Open AsyncAPI Document in Studio",
+      "description": "Opens a given AsyncAPI document with the AsyncAPI Studio",
+      "mode": "view"
+    },
+    {
+      "name": "open-in-studio-clipboard",
+      "title": "Open AsyncAPI Document in Studio",
+      "subtitle": "From clipboard",
+      "description": "Opens the AsyncAPI document from clipboard with the AsyncAPI Studio",
+      "mode": "no-view"
+    }
+  ],
+  "dependencies": {
+    "@asyncapi/parser": "^1.14.1",
+    "@raycast/api": "^1.28.0"
```

```suggestion
    "@raycast/api": "^1.30.1"
```
Could be cool to use the newest version too 😄

---

Makes sense as it is required to work 👍